conf/post_deploy_actions.bash

```diff
@@ -11,29 +11,27 @@ cd `dirname $0`/..
 # for example xapian. If using these you might want to add the
 # '--system-site-packages' argument.
 virtualenv_args=""
-virtualenv_dir='../virtualenv-repository'
+virtualenv_dir='../venv'
 virtualenv_activate="$virtualenv_dir/bin/activate"
 if [ ! -f "$virtualenv_activate" ]
 then
-    virtualenv $virtualenv_args $virtualenv_dir
+    python3 -m venv $virtualenv_dir  --without-pip
 fi
 source $virtualenv_activate
 
 # Upgrade pip to a secure version
+curl -L -s https://bootstrap.pypa.io/pip/3.5/get-pip.py | python3
```

Comment:
Current pattern is to install `pip` with the virtualenv, and skip the curl. We do sometimes install `wheel` here, though, so perhaps add that.

Reply:
Is there a good one to copy somewhere?

Reply:
Yeah, check out the one in [MapIt Global](https://github.com/mysociety/global.mapit.mysociety.org/blob/master/conf/pre_deploy_actions.bash) as an example perhaps.

Reply:
I've copied the split between pre_ and post_ deploy in the mapit global. Should now be closer to right.